README.md

```diff
@@ -168,29 +168,83 @@ In addition to the examples above, please refer to the documentation for tutoria
 
 ## Updates
 
-### v2.1.0 (2Q24)
+### v2.1.0 (3Q24)
 
-TBD
+[Product name, version X.X] does not include the latest functional and security updates. [Product name, Version X.X] is targeted to be released in [Month 20XX] and will include additional functional and security updates. Customers should update to the latest version as it becomes available.
```

Comment:
```suggestion
OpenVINO™ Training Extensions, version 2.1.0 does not include the latest functional and security updates. OpenVINO™ Training Extensions, version 2.2.0 is targeted to be released in [Month 20XX] and will include additional functional and security updates. Customers should update to the latest version as it becomes available.
```
It seems typo

Reply:
i've just forgot to delete the template statement. in the next line, we have correct statement inside of `notes`